src/db.cpp

```diff
@@ -21,6 +21,19 @@ bool DB<T, U>::put(Key<T> key, Value<U> value) {
     return true;
 }
 
+template<class T, class U>
+bool DB<T, U>::put(int key, int value) {
```

Comment:
why doesn't this take generic T key and generic U value?

Reply:
bc this is a function that is meant to put an `int` key and `int` value. It's just a helper function instead of creating the Key and Value, then the Entry, then putting. This streamlines it.